src/app/screens/Send/index.tsx

```diff
@@ -91,6 +97,13 @@ function Send() {
             },
           },
         });
+      } else if (
+        isAlbyOAuthAccount(auth.account?.connectorType) &&
```

Comment:
Why are other connectors not supported?

Reply:
From Rene: this endpoint is only available for logged-in users (it is part of the OAuth API)

Reply:
can this actually be a function on the `auth` itself?